server/controllers/connection-status.js

```diff
@@ -0,0 +1,44 @@
+const {
+    queryDynamoByChannel,
+    updateConnectionStatus,
+} = require('../lib/dynamoDao');
+
+// TO BE USED LATER AS SOME KIND OF TYPESAFE ENUM VALIDATION
+const CONNECTION_STATUS = {
+    ACTIVE: 'active',
+    INACTIVE: 'inactive',
+    STARTING: 'starting',
+};
+const ConnectionStatusController = {
+    getConnectionStatus(req, res) {
+        const channelId = req.params.channelId;
+        console.log({ channelId });
+
+        queryDynamoByChannel(channelId)
+            .then((item) => {
+                if (!item) {
+                    res.status(404).json({
+                        message: 'No items found for provided channelId',
+                    });
+                } else {
+                    res.json({ item });
+                }
+            })
+            .catch((err) => {
+                console.log(err);
+                res.status(500).json({ err });
+            });
+    },
+    updateConnectionStatus(channelId, connectionStatus) {
```

Comment:
I don't like that the function name here is shadowed by the name in the dynamo lib. This causes ambiguity and harder to trace. Either the method name in this controller needs to change, or the one in the dynamo lib needs to.